### ISSUE TITLE
undefined: robotgo.SaveCapture

### ISSUE THREAD

**Issue by Norman-Normandy** [go-vgo/robotgo#668]
- Robotgo version (or commit ref): Latest [63eb594](https://github.com/go-vgo/robotgo/commit/63eb594ea5d52fb002928dbf140c36d14092720d)
- Go version: go version go1.22.5 linux/amd64
- Gcc version: gcc (GCC) 13.2.0
- Operating system and bit: Void Linux
- Resolution: 1440P
- Can you reproduce the bug at [Examples](https://github.com/go-vgo/robotgo/blob/master/examples/main.go):
  - [x] Yes (provide example code)
  - [ ] No
  - [ ] Not relevant
- Provide example code:

```
package main

import (
  "fmt"
  "math/rand"

  "github.com/go-vgo/robotgo"
  "github.com/vcaesar/gcv"
  "github.com/vcaesar/bitmap"
)

func main() {
  opencv()
}

func opencv() {
  name := "test.png"
  name1 := "test_001.png"
  robotgo.SaveCapture(name1, 10, 10, 30, 30)
  robotgo.SaveCapture(name)

  fmt.Print("gcv find image: ")
  fmt.Println(gcv.FindImgFile(name1, name))
  fmt.Println(gcv.FindAllImgFile(name1, name))

  bit := bitmap.Open(name1)
  defer robotgo.FreeBitmap(bit)
  fmt.Print("find bitmap: ")
  fmt.Println(bitmap.Find(bit))

  // bit0 := robotgo.CaptureScreen()
  // img := robotgo.ToImage(bit0)
  // bit1 := robotgo.CaptureScreen(10, 10, 30, 30)
  // img1 := robotgo.ToImage(bit1)
  // defer robotgo.FreeBitmapArr(bit0, bit1)
  img := robotgo.CaptureImg()
  img1 := robotgo.CaptureImg(10, 10, 30, 30)

  fmt.Print("gcv find image: ")
  fmt.Println(gcv.FindImg(img1, img))
  fmt.Println()

  res := gcv.FindAllImg(img1, img)
  fmt.Println(res[0].TopLeft.Y, res[0].Rects.TopLeft.X, res)
  x, y := res[0].TopLeft.X, res[0].TopLeft.Y
  robotgo.Move(x, y-rand.Intn(5))
  robotgo.MilliSleep(100)
  robotgo.Click()

  res = gcv.FindAll(img1, img) // use find template and sift
  fmt.Println("find all: ", res)
  res1 := gcv.Find(img1, img)
  fmt.Println("find: ", res1)

  img2, _, _ := robotgo.DecodeImg("test_001.png")
  x, y = gcv.FindX(img2, img)
  fmt.Println(x, y)
}
```

```Go

```
- Log gist:

## Description

Basically trying run all example snippets to see that the library works as intended. However on this one example for OpenCV it fails with the error: 'undefined: robotgo.SaveCapture'
Being repeated twice. I'm certain I have all dependencies installed and imported all relevant packages. The other examples work fine.
...

**Comment by 117503445** [comment]
```go
img := robotgo.CaptureImg(10, 10, 30, 30)
if err = robotgo.Save(img, name); err != nil {
    panic(err)
}
```

works
